crates/dprint/src/cli/run_cli.rs

```diff
@@ -665,90 +705,99 @@ fn resolve_plugins<TEnvironment: Environment>(
     return Ok(plugins);
 }
 
-fn resolve_file_paths(config: &ResolvedConfig, args: &CliArgs, environment: &impl Environment) -> Result<Vec<PathBuf>, ErrBox> {
+fn get_and_resolve_file_paths(config: &ResolvedConfig, args: &CliArgs, environment: &impl Environment) -> Result<Vec<PathBuf>, ErrBox> {
```

Comment:
Pulled this out into a new method, just so that we can return the file patterns / resolved paths and store them.